app/components/HeaderNotifications/index.tsx

```diff
@@ -118,7 +118,7 @@ const NotificationsDropdown = () => {
       }}
       closeOnContentClick
       triggerComponent={
-        <Icon.Badge
+        <BadgeIcon
```

Comment:
Why is this new

Reply:
`Icon.Badge` wasn't possible anymore with the `forwardRef` in `Icon`.